.changeset/big-hornets-smell.md

```diff
@@ -0,0 +1,14 @@
+---
+'@openfn/language-primero': minor
+---
+
+- Add http generic helpers with `GET`, `POST`, and `PATCH`.
```

Comment:
I think we should say:
- Added a new http namespace, containing HTTP helpers for `get()` `post()` and `patch()`
- Mention the difference between these new helpers and the existing ones. Like, why are we doing this, and what should users prefer going forward?


Btw I think we need to stop using the phrase "generic HTTP helpers"... because none of them are generic and it's misleading :)